packages/ui/src/fields/Textarea/types.ts

```diff
@@ -11,11 +20,12 @@ export type TextAreaInputProps = {
   readonly beforeInput?: MappedComponent[]
   readonly className?: string
   readonly description?: StaticDescription
-  readonly descriptionProps?: Record<string, unknown>
-  readonly errorProps?: Record<string, unknown>
+  readonly descriptionProps?: FieldDescriptionClientProps<MarkOptional<TextareaFieldClient, 'type'>>
+  readonly errorProps?: FieldErrorClientProps<MarkOptional<TextareaFieldClient, 'type'>>
+  readonly field?: MarkOptional<TextareaFieldClient, 'type'>
   readonly inputRef?: React.RefObject<HTMLInputElement>
   readonly label: StaticLabel
-  readonly labelProps?: Record<string, unknown>
+  readonly labelProps?: FieldLabelClientProps<MarkOptional<TextareaFieldClient, 'type'>>
```

Comment:
Same here and for `field` we could use `TextFieldClientWithoutType` - no strong preference though